package.json

```diff
@@ -152,7 +153,7 @@
     "traverse": "^0.6.6",
     "treat": "^2.0.4",
     "tree-kill": "^1.2.1",
-    "typescript": ">=4.1.3 <4.5.0",
+    "typescript": "^4.5.0",
```

Comment:
This [goes](https://github.com/seek-oss/sku/blob/b03adb48391fafb44fd7cfa8c34d73dd952f5d4f/yarn.lock#L16975-L16976) straight to latest, `4.8.4`. Do we want to use `~4.5.0` to stay on `4.5.x`?

Reply:
Skuba does this, though I'm not exactly sure of the benefit in the case of sku. If anything, I feel like we'd want to restrict the upper limit, since typescript doesn't adhere to semver, just in case something breaks, but I'm not sure about restricting to a specific minor version just yet.